nav2_behavior_tree/plugins/decorator/distance_controller.hpp

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2018 Intel Corporation
```

Comment:
Hmm, I'm ok with it, but why not keeping everything on the .cpp file as it is done with the rest of plugins?

Reply:
The plugins are compiled as shared libraries so I couldn't figure out a way to include them in the unit tests. The easiest solution I could think of was to write a header file so I can make that available to the tests.

Reply:
All the plugins will have to be divided into .hpp files if we want to write some unit tests (correct me if I'm wrong)

Reply:
Why not creating a dummy behaviour tree that loads and tests each plugin? The nodes should contain simple enough functionality to make unit tests redundant.
How is this done in BehaviourTree.cpp? And what about the recovery plugins?

I think it would get a bit dirty to have to include header files for each plugin, specially when they are all written in less than 100 lines.